source/vibe/core/drivers/libasync.d

```diff
@@ -481,6 +478,8 @@ final class LibasyncFileStream : FileStream {
 
 	void close()
 	{
+		if (m_task != Task())
+			getDriverCore().resumeTask(m_task, new Exception("The file was closed during an operation"));
```

Comment:
This should probably be `yieldAndResultTask` instead, as it will be quite likely that `close` is called from within a task. It might also make sense to move this further down, so that when that task continues and queries `isOpen`, it will get `false` instead of `true`.